docs/openapi-docs/src/test/resources/expected_coproduct.yml

```diff
@@ -12,11 +12,13 @@ paths:
           content:
             application/json:
               schema:
-                oneOf:
-                  - $ref: '#/components/schemas/Person'
-                  - $ref: '#/components/schemas/Organization'
+                $ref: '#/components/schemas/Entity'
 components:
   schemas:
+    Entity:
+      oneOf:
+        - $ref: '#/components/schemas/Person'
+        - $ref: '#/components/schemas/Organization'
```

Comment:
that's valid for openapi?

Reply:
I checked it using openapi online editor and it didn't complain